docs/tutorials/podman_pasta_ipv6_with_snid_ipv4.md

```diff
@@ -0,0 +1,761 @@
+
+```
+services:
+  whoami-caddy:
+    image: caddy:latest
+    #image: lucaslorentz/caddy-docker-proxy:2.9-alpine
+    pull_policy: "missing"
+    container_name: whoami-caddy
+    restart: "unless-stopped"
+    security_opt:
+      - no-new-privileges:true
+      - label=type:container_runtime_t
+    network_mode: "pasta:--ipv6-only,-t,${APPLICATION_IPV6_ADDRESS}/80,-t,${APPLICATION_IPV6_ADDRESS}/443,-u,${APPLICATION_IPV6_ADDRESS}/443"
+    volumes:
+ #     - /run/user/1001/podman/podman.sock:/var/run/docker.sock:rw,z
+      - ./Caddyfile:/etc/caddy/Caddyfile:ro,z
+      - ~/containers/data/whoami-caddy:/data:rw,z
+      - ~/containers/log/whoami-caddy:/var/log:rw,z
+      - ~/containers/config/whoami-caddy:/config:rw,z
+      - ~/containers/certificates/letsencrypt:/certificates:ro,z
+    environment:
+      - CADDY_DOCKER_CADDYFILE_PATH=/etc/caddy/Caddyfile
+
+  # Proxy to container
+  whoami-application:
+    image: traefik/whoami
+    pull_policy: "missing"
+    container_name: whoami-application
+    restart: "unless-stopped"
+    network_mode: "service:whoami-caddy"
+    environment:
+      - WHOAMI_PORT_NUMBER=8080
+```
+
+> **Warning**  
+> 
+> Netstat is deprecated. `netstat -an | grep -i listen` will NOT return the correct IPv6 Addresses in most cases, because it truncates the Output. `netstat -Wan | grep -i listen` will return the full IPv6 Address wthout truncating it, but you should really be using `ss -nlt6` instead.
+
+# Caddy Proxy Configuration
+For Simple Configurations of Applications and automatically generating a SSL Certificate using Letsencrypt with the HTTP(S) Challenge, one can just define `command` within the `compose.yml` File to something like:
+```
+command: reverse-proxy --from application01.MYDOMAIN.TLD --to 'http//[::1]:8080'
+```
+
+For semi-automated Setups, one can use the `lucaslorentz/caddy-docker-proxy` Docker Image, which allows to set Caddy Options directly within the `compose.yml` File.
+
+I am NOT used to this Syntax, so I am just using the Caddyfile, at least for now.
+This is also due to the Fact that I am self-managing the Letsencrypt Certificates using `certbot` and distributing them across my Infrastructure.
+
+Your mileage may vary :).
+
+```
+# Example and Guide
+# https://caddyserver.com/docs/caddyfile/options
+
+# General Options
+{
+    # Debug Mode
+    debug
+
+    # Ports Configuration
+    #http_port 80
+    #https_port 443
+
+    # TLS Options
+    auto_https disable_certs
+
+    # Default SNI
+    default_sni MYDOMAIN.TLD
+}
+
+
+localhost {
+	reverse_proxy /api/* localhost:9001
+}
+
+application01.MYDOMAIN.TLD {
+        tls /certificates/MYDOMAIN.TLD/fullchain.pem /certificates/MYDOMAIN.TLD/privkey.pem
+        log {
+		output file /var/log/application01.MYDOMAIN.TLD/access.json {
+			roll_size 100MiB
+			roll_keep 5000
+			roll_keep_for 720h
+            roll_uncompressed
+		}
+        format json
+	}
+
+    reverse_proxy http://[::1]:8080
+}
+```
```

Comment:
can be removed completely, if the compose file makes use of `command: caddy reverse-proxy ...`

Reply:
Keyword is **IF** (with the certificates management using certbot, custom logging, etc, as explained before).

Anyways, I split both the compose part and the Caddy Configuration into two Subsections:
1. Using Default Caddy Configuration
2. Using Customized Caddy Configuration

I implemented your short(est) version in the first subsection (which probably fits most users), while keeping the one I currently have (and use on my Server) as second.